### PR TITLE
init wdio-lwc-service package

### PR DESCRIPTION
**Pull request by Templarian**
Goal: Move from https://github.com/salesforce/wdio-lwc-service to this repo.

### CODE REVIEW

**Comment by Templarian** [comment]
@trevor-bliss Good catch, removed those two files. 👍

**Comment by Templarian** [comment]
@trevor-bliss Assuming this passes the CI today we can be merged (I don't have write to this repo). Thanks. 😃

**Comment by trevor-bliss** [comment]
@Templarian Yeah definitely. Let me dig around the CI and see if I can figure out what's going on...

File: packages/wdio-lwc-service/examples/basic/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "description": "Example of LWC standalone app using wdio-lwc-service",
+  "main": "index.js",
+  "author": "Reinier Guerra",
+  "license": "MIT",
+  "scripts": {
+    "start": "run-p watch server",
+    "test:wdio": "wdio wdio.conf.js",
+    "server": "node server.js",
+    "watch": "rollup -c -w",
+    "lint": "eslint src --ext .js --max-warnings 0 --cache"
```
Comment (trevor-bliss) [inline packages/wdio-lwc-service/examples/basic/package.json]: Need `eslint` dep. Currently, I get this:
```
basic $ yarn lint
yarn run v1.17.3
$ eslint src --ext .js --max-warnings 0 --cache
/bin/sh: eslint: command not found
error Command failed with exit code 127.
```

**Comment by trevor-bliss** [comment]
Updated a setting in CircleCI, closing/opening this to test.

File: packages/wdio-lwc-service/examples/basic/package.json (hunk 1 above)
Comment (Templarian) [reply packages/wdio-lwc-service/examples/basic/package.json]: Fixed. 😃

**Pull request merged**